orchagent/fdborch.cpp

```diff
@@ -1306,7 +1546,7 @@ bool FdbOrch::removeFdbEntry(const FdbEntry& entry, FdbOrigin origin)
     {
         SWSS_LOG_ERROR("FdbOrch RemoveFDBEntry: Failed to remove FDB entry. mac=%s, bv_id=0x%" PRIx64,
                        entry.mac.to_string().c_str(), entry.bv_id);
-        task_process_status handle_status = handleSaiRemoveStatus(SAI_API_FDB, status); //FIXME: it should be based on status. Some could be retried. some not
+        task_process_status handle_status = handleSaiRemoveStatus(SAI_API_FDB, status); //Address me: it should be based on status. Some could be retried. some not
```

Comment:
Same here

Reply:
@prsunny, @gechiang  : This code with FIXME comment exist befored and I was asked to modify by gechiang to 'Address me'. 
Please confirm if this needs to be reverted back to ok to keep as it is .?

Reply:
I think we can stick to "FIXME" as it is used in other parts of file. May be the comment was for newly added code. @gechiang, can we reuse the existing comments?

Reply:
@prsunny if we revert back then we will start seeing the LGTM raised alert due to observing "FIXME"... if you think this is fine, we can revert it back... Also, is the new best practice for new code to avoid making comments with "FIXME"?

Reply:
@gechiang , @prsunny  Thanks. I have reverted back